x-pack/plugins/triggers_actions_ui/server/routes/config.test.ts

```diff
@@ -54,6 +54,7 @@ describe('createConfigRoute', () => {
         isUsingSecurity: true,
         maxScheduledPerMinute: 10000,
         minimumScheduleInterval: { value: '1m', enforce: false },
+        run: { alerts: { max: 1000 }, actions: { max: 100000 } },
```

Comment:
These are set here as the new config typing expects the `run` key, but the output of the route is unchanged